querydsl-sql/src/main/java/com/querydsl/sql/OracleTemplates.java

```diff
@@ -135,7 +135,7 @@ public OracleTemplates(char escape, boolean quote) {
 
         add(Ops.DateTimeOps.TRUNC_YEAR, "trunc({0}, 'year')");
         add(Ops.DateTimeOps.TRUNC_MONTH, "trunc({0}, 'month')");
-        add(Ops.DateTimeOps.TRUNC_WEEK, "trunc({0}, 'w')");
+        add(Ops.DateTimeOps.TRUNC_WEEK, "trunc({0}, 'day')");
```

Comment:
This looks like a typo. Please see [official Oracle docs](https://docs.oracle.com/cd/B19306_01/server.102/b14200/functions230.htm#i1002084)

Reply:
Seems you're right based on the documentation. Most databases use "w" or "week" here. What a poor choice on Oracle's side to use "day" here...

Reply:
I've changed this to 'iw', which always truncates to ISO week start (Monday). Oracle docs are extremely confusing, but my testing shows that 'iw' always truncates to Monday regardless of the territory config (unlike 'day')